src/pybert/pybert.py

```diff
@@ -615,11 +606,11 @@ def log(self, msg, alert=False, exception=None):
         """Log a message to the console and, optionally, to terminal and/or
         pop-up dialog."""
         _msg = msg.strip()
-        txt = f"\n[{datetime.now()}]: PyBERT: {_msg}\n"
+        txt = f"[{datetime.now()}]: PyBERT: {_msg}"
         if self.debug:
             ## In case PyBERT crashes, before we can read this in its `Console` tab:
             print(txt, flush=True)
-        self.console_log += txt
+        self.console_log += txt + "\n"
```

Comment:
Wow! I really was in love w/ carriage returns, here, wasn't I? ;-)
Thanks for cleaning this up! :)

Reply:
Three when printing and two in the log. :) Can never have enough!